README.md

```diff
@@ -113,8 +113,9 @@ $ docker run -p 8080:80 -e SERVER_NAME=myhost my-modsec
 
 | Name     | Description|
 | -------- | ------------------------------------------------------------------- |
-| ACCESSLOG | A string value indicating the location of the custom log file (Default: `/var/ log/apache2/access.log`) |
-| APACHE_ALWAYS_TLS_REDIRECT | A string value indicating if http should redirect to https (Allowed values: `on`, `off`. Default: `off`) | 
+| ACCESSLOG | A string value indicating the location of the custom log file (Default: `/var/log/apache2/access.log`) |
+| APACHE_ALWAYS_TLS_REDIRECT | A string value indicating if http should redirect to https (Allowed values: `on`, `off`. Default: `off`) |
+| APACHE_LOGFORMAT | A string value indicating the LogFormat that apache should use. Default: `'"%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\""'` (combined). Tip: use single quotes outside your double quoted format string. \ |
```

Comment:
Not really necessary but it would be nice to have separate log formats for the different log files (access, error, transfer), especially for the error log which has a separate log format command `ErrorLogFormat`.

Reply:
We can create a new issue and track it there.

Reply:
https://github.com/coreruleset/modsecurity-docker/issues/127

Reply:
Thanks.